FAQ.md

```diff
@@ -529,6 +529,40 @@ The `create_time` of tables in the `information_schema` is the creation time.
 
 When TiDB is executing a SQL statement, the query will be `EXPENSIVE_QUERY` if each operator is estimated to process over 10000 pieces of data. You can modify the `tidb-server` configuration parameter to adjust the threshold and then restart the `tidb-server`.
 
+#### How to control or change the execution priority of SQL commits?
+
+TiDB has the following high priority and low priority syntax:
+
+- HIGH_PRIORITY: this statement has a high priority, that is, TiDB gives priority to this statement and executes it first.
+
+- LOW_PRIORITY: this statement has a low priority, that is, TiDB reduces the priority of this statement during the execution period.
+
+You can combine the above two parameters with the DML language of TiDB to use them. For usage details, see [TiDB DML](sql/dml.md). For example:
```

Comment:
DML (Data Manipulation Language) already includes "language", so delete the "language" after it.